san/sanbase_graphql.py

```diff
@@ -307,27 +166,6 @@ def emerging_trends(idx, **kwargs):
     return query_str
 
 
-def top_social_gainers_losers(idx, **kwargs):
```

Comment:
You should check also to remove in the README. For example this one is present there.

Reply:
Done.